### PR TITLE
IOTEX-275 Get rid of old genesis completely

### PR DESCRIPTION
**Pull request by zjshen14**
Skip the tests that require start subchain action in genesis. Fix them later.

### CODE REVIEW

**Comment by zjshen14** [comment]
Verified in the real cluster

File: action/protocol/poll/protocol.go [hunk 1]
```diff
@@ -58,18 +57,13 @@ type lifeLongDelegatesProtocol struct {
 }
 
 // NewLifeLongDelegatesProtocol creates a poll protocol with life long delegates
-func NewLifeLongDelegatesProtocol(delegates []genesis.Delegate) Protocol {
+func NewLifeLongDelegatesProtocol(addrs []address.Address) Protocol {
```
Comment (CoderZhi) [inline action/protocol/poll/protocol.go]: with votes and reward addresses, ppl could easily use lifelongdelegatesprotocol to start a consortium chain with fixed portion for each participate, or some other purpose. no hurt to keep this

File: action/protocol/execution/protocol_test.go [hunk 2]
```diff
@@ -14,6 +14,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/iotexproject/iotex-core/pkg/unit"
```
Comment (CoderZhi) [inline action/protocol/execution/protocol_test.go]: group

File: blockchain/blockchain_test.go [hunk 3]
```diff
@@ -13,6 +13,8 @@ import (
 	"sync"
 	"testing"
 
+	"github.com/iotexproject/iotex-core/pkg/unit"
```
Comment (CoderZhi) [inline blockchain/blockchain_test.go]: group

File: blockchain/genesis/genesis.go [hunk 4]
```diff
@@ -8,10 +8,13 @@ package genesis
 
 import (
 	"flag"
+	"io/ioutil"
 	"math/big"
 	"sort"
 	"time"
 
+	"github.com/iotexproject/iotex-core/pkg/hash"
```
Comment (CoderZhi) [inline blockchain/genesis/genesis.go]: group

**Pull request merged**